src/Nox.Generator/Infrastructure/Persistence/DbContext.template.cs

```diff
@@ -116,17 +117,20 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
         modelBuilder.ForEntitiesOfType<IEntityConcurrent>(
             builder => builder.Property(nameof(IEntityConcurrent.Etag)).IsConcurrencyToken());
-
-        {{- for entityAtt in enumerationAttributes #Setup Entity Enumerations}}
-        {{- for enumAtt in entityAtt.Attributes}}
-            ConfigureEnumeration(modelBuilder.Entity("{{codeGeneratorState.DomainNameSpace}}.{{enumAtt.EntityNameForEnumeration}}"));
-            {{- if enumAtt.Attribute.EnumerationTypeOptions.IsLocalized}}
-            var enumLocalizedType = _clientAssemblyProvider.GetType("{{codeGeneratorState.DomainNameSpace}}.{{enumAtt.EntityNameForLocalizedEnumeration}}")!;
-            var enumType = _clientAssemblyProvider.GetType("{{codeGeneratorState.DomainNameSpace}}.{{enumAtt.EntityNameForEnumeration}}")!;
-            ConfigureEnumerationLocalized(modelBuilder.Entity(enumLocalizedType), enumType, enumLocalizedType);
-            {{- end }}
-        {{- end }}
-        {{- end }}
+    }
+    
+    private void ConfigureEnumeratedAttributes(ModelBuilder modelBuilder, Entity entity)
+    {
+        foreach(var enumAttribute in entity.Attributes.Where(attribute => attribute.Type == NoxType.Enumeration))
```

Comment:
You already have the Enumeration attributes and their names as an input parameter "enumerationAttributes", please re-use that one.

Reply:
As we discussed, keeping this as because here we need instance of an entity and attribute to be able to access enum type options and not just names.